features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/h2.sql

```diff
@@ -64,14 +64,28 @@ CREATE TABLE IF NOT EXISTS DM_PROFILE_OPERATION (
     DM_OPERATION (ID) ON DELETE NO ACTION ON UPDATE NO ACTION
 );
 
-CREATE TABLE IF NOT EXISTS DM_DEVICE_OPERATION_MAPPING (
+CREATE TABLE IF NOT EXISTS DM_ENROLMENT (
     ID INTEGER AUTO_INCREMENT NOT NULL,
     DEVICE_ID INTEGER NOT NULL,
+    OWNER VARCHAR(50) NOT NULL,
+    OWNERSHIP VARCHAR(45) NULL DEFAULT NULL,
+    STATUS VARCHAR(50) NULL,
+    DATE_OF_ENROLMENT TIMESTAMP NULL DEFAULT NULL,
+    DATE_OF_LAST_UPDATE TIMESTAMP NULL DEFAULT NULL,
+    TENANT_ID INT NOT NULL,
+    PRIMARY KEY (ID),
+    CONSTRAINT fk_dm_device_enrolment FOREIGN KEY (DEVICE_ID) REFERENCES
+    DM_DEVICE (ID) ON DELETE NO ACTION ON UPDATE NO ACTION
+);
+
+CREATE TABLE IF NOT EXISTS DM_ENROLMENT_OPERATION_MAPPING (
+    ID INTEGER AUTO_INCREMENT NOT NULL,
+    ENROLMENT_ID INTEGER NOT NULL,
     OPERATION_ID INTEGER NOT NULL,
     STATUS VARCHAR(50) NULL,
     PRIMARY KEY (ID),
-    CONSTRAINT fk_dm_device_operation_mapping_device FOREIGN KEY (DEVICE_ID) REFERENCES
-    DM_DEVICE (ID) ON DELETE NO ACTION ON UPDATE NO ACTION,
+    CONSTRAINT fk_dm_device_operation_mapping_device FOREIGN KEY (ENROLMENT_ID) REFERENCES
+    DM_ENROLMENT (ID) ON DELETE NO ACTION ON UPDATE NO ACTION,
     CONSTRAINT fk_dm_device_operation_mapping_operation FOREIGN KEY (OPERATION_ID) REFERENCES
     DM_OPERATION (ID) ON DELETE NO ACTION ON UPDATE NO ACTION
```

Comment:
Why ON DELETE NO ACTION? shouldn't this be ON DELETE CASCADE?
Why ON UPDATE NO ACTION? shouldn't this be ON UPDATE CASCADE?